src/Aspire.Dashboard/Model/ResourceDataKeys.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Dashboard.Model;
+
+public static class ResourceDataKeys
```

Comment:
It might be nice to link to docs around the grpc request these are associated with

Reply:
These keys are associated with specific resource types, not so much with specific gRPC requests. For example, if you want all resources, and have all kinds of resources in your system, you'll potentially see all these keys across the set of data (returned via the single gRPC service call).